docs/contributing/contrib_pull_request_acceptance_checklist.rst

```diff
@@ -0,0 +1,78 @@
+.. _contrib_pull_request_acceptance_checklist:
+
+
+Acceptance Checklist for "contrib" Pull Requests
+================================================
+
+This acceptance checklist applies to "contrib" Pull Requests - PRs that make modifications only in ``/contrib`` directory of the repo which contains community contributed Expectations.
+
+
+Both contributors and reviewers should follow this checklist:
+
+* Scope of the PR.
+    * A PR may introduce a new Expectation. The minimal acceptable version of a new Expectation is a module that contains a new class with the name of the Expectation, a Docstring describing the behavior, and examples/tests. The class may leave all methods unimplemented. Metric classes required by the new Expectation may be left undefined/unimplemented. This minimal contribution can be viewed as a "RFE" - Request for Expectation.
+    * If the PR modifies existing Expectations and/or Metrics, check that the changes are an improvement and are not destructive. The contributor should not remove someone else' prior work without a clear reason.
+
+* Basic class/module structure and naming of new Expectation and Metric classes
+    * One new Expectation class per module.
+    * The modules are placed in a subdirectory of `/contrib/experimental/expectations` that makes sense for the new Expectation
+    * The new Expectation classes follow CamelCase convention. Module names are snake_case versions of the Expectation class names.
+    * Metric classes that implement the new Metrics the Expectation depends on are implemented in the same module.
+    * Expectation class names summarize the nature of the expectation - this is a judgement call.
+    * New Expectation and Metric classes extend the appropriate core classes (e.g., new column map expectations extend ColumnMapExpectation)
+
+* Docstring
+    * Docstring must contain a legible description of the functionality of the new Expectations.
+
+* Library metatada
+    * this is the information required to add an Expectation to the Public Expectation Gallery. We expect the contributor to fill out this section, but we will "pick up the slack" if needed. The module containing a new Expectation must have a comment that looks as follows:
```

Comment:
```suggestion
    * The `library_metadata` is the information required to add an Expectation to the Public Expectation Gallery. Contributors, please complete as much of the section as possible. Reviewers, please complete any remaining fields to avoid extra asks to contributors. Adapt the metadata example below as appropriate for the new contribution.
```